src/components/AppMap.vue

```diff
@@ -241,6 +241,8 @@
         <b-checkbox switch v-model="showSafeAreas" @change="onShowSafeAreas">Enemy Non-Search Areas (Safe Zones)</b-checkbox>
         <b-checkbox switch v-model="showMapUnitGrid" @change="onShowMapUnitGridChanged">Show map unit grid</b-checkbox>
         <b-checkbox switch v-model="showCastleAreas" @change="onShowCastleAreas">Show Hyrule Castle Interior</b-checkbox>
+        <b-checkbox switch v-model="showBaseMap" @change="onShowBaseMap">Show Base Map</b-checkbox>
+        <b-checkbox switch v-model="showReferenceGrid" @change="onShowReferenceGrid">Show Map Reference Grid</b-checkbox>
```

Comment:
I feel like "reference grid" is not a super obvious name but I don't really have a better name to suggest...

Reply:
Do any of these work:
- Show Undiscovered Tower Map
- Show Undiscovered Regional Map
- Show Undiscovered Map
- Show Pre-Tower Activation Map

Reply:
Hmm I think we do need "grid" somewhere in the description... The option also controls whether region outlines are shown, so maybe "Show region outlines and grid"? (Also we should make the sentence casing more consistent)

Reply:
I like "Show region outlines and grid" 

How about also adding a `title` text giving further explanation
```Display the map as it exists prior to Tower activation, with only region outlines and grid markers```
Added a title text needs a `div` inside the `b-checkbox`

I lower-cased all of the text in that section with only the first word capitalized, along with Hyrule (sentence capitalization?)

Reply:
Adding a tooltip description of the option sounds good to me, but I think the proposed title is a bit misleading; wouldn't you need to enable "show region outlines and grid" **and** disable "show base map" to get the described effect?

Sentence capitalisation sounds good - that's what BotW does with longer toggles in the Options screen.